src/Symfony/Component/Cache/CacheInterface.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Cache;
+
+use Psr\Cache\CacheItemInterface;
+
+/**
+ * Gets and stores items from a cache.
+ *
+ * @author Nicolas Grekas <p@tchwork.com>
+ */
+interface CacheInterface
+{
+    /**
+     * @param callable(string, CacheItemInterface):mixed $callback Should return the computed value for the given key/item
+     *
+     * @return mixed The value corresponding to the provided key
+     */
+    public function get(string $key, callable $callback);
+
+    /**
+     * @param callable(string, CacheItemInterface):mixed $callback Should return the computed value for the given key/item
+     *
+     * @return iterable The values corresponding to the provided keys
+     */
+    public function getMultiple(iterable $keys, callable $callback): iterable;
```

Comment:
How about making tagging also support iterable?

https://github.com/symfony/symfony/blob/ccb0e8479901358e7a7cb0c98d3b58990c619386/src/Symfony/Component/Cache/CacheItem.php#L112

Reply:
good idea, done

Reply:
>$tags A tag or array of tags

iterable of tags